compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/IntermediateTextInputUIView.uikit.kt

```diff
@@ -87,7 +87,7 @@ internal class IntermediateTextInputUIView(
                 hideEditMenu()
             }
         }
-    var keyboardEventHandler: KeyboardEventHandler? = null
+    var onPresses: (Set<*>) -> Unit = NoOpOnPresses
```

Comment:
Are presses can be done from another source, different from the attached keyboard?
I would keep `keyboard` naming part in API to have more context when reading the code.

Reply:
Yes, hardware keyboard too, afaik.
Anyway, a good point, will add it.